app/src/main/java/com/architect/coders/mu8/MainActivity.kt

```diff
@@ -2,11 +2,13 @@ package com.architect.coders.mu8
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import com.architect.coders.mu8.data.DummyClass
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        val damy = DummyClass("Alicia")
```

Comment:
Remove this usage from the presentation module, it is ok for testing but let's not push something we don't want to have.